client/my-sites/plans/ecommerce-trial/upgrade-confirmation/confirmation-task/style.scss

```diff
@@ -1,4 +1,32 @@
 .confirmation-task__card {
+	--subtitle-font-size: 0.8125rem;
+	border-radius: 2px;
+	box-shadow: 0 0 0 1px var(--color-neutral-5);
+	cursor: pointer;
+
+	&:hover {
+		box-shadow: 0 0 0 2px var(--color-accent-50);
+	}
```

Comment:
Might it be simpler to implement these as a `border` instead of `box-shadow`? (The Figma designs use `border`.)

Reply:
I ended up using box-shadow here because the card component already uses it, and it also solves the box-sizing problem because we have 2 different border-widths.  
I'm merging as is since you also don't consider it blocking.